pkg/rpcclient/wsclient.go

```diff
@@ -678,7 +679,7 @@ func (c *WSClient) ReceiveBlocks(flt *neorpc.BlockFilter, rcvr chan<- *block.Blo
 	}
 	params := []interface{}{"block_added"}
 	if flt != nil {
-		params = append(params, *flt)
+		params = append(params, *flt.Copy())
```

Comment:
I think it's not enough. Three lines below we add the same user's filter to the receiver, and after that client uses receiver's filter to filter out events. Thus, the user can still change something in filter and break the receiver.Let's provide copy to the receiver as far.

Reply:
Fixed!